README.md

```diff
@@ -47,17 +40,23 @@ The original values can be found on https://docs.openmicroscopy.org/omero/5.6.0/
 Certificates will be stored under `{omero.data.dir}/certs` by default.
 Set `omero.glacier2.IceSSL.DefaultDir` to change this.
 
-If you see a warning message such as
+For full information see the output of:
 ```
-Can't load ./.rnd into RNG
+omero certificates --help
 ```
-it should be safe to ignore.
 
-For full information see the output of:
+## Upgrading
+
+Since version 0.3.0 this plugin uses portable RFC 4514 (supercedes RFC 2253) formatted strings for the `omero.certificates.owner` configuration option.  If you have ran `omero certificates` before you may have OpenSSL command line formatted strings in your configuration that should be updated before you can run `omero certificates` again.  In most cases this means taking a string such as `/L=OMERO/O=OMERO.server` and reformatting it to `L=OMERO,O=OMERO.server`; remove the leading `/` and replace separator `/`'s with `,`'s.
```

Comment:
Quick note before I need to head out: there is a place in omero-py for upgrading properties in case we want to make use of it. (And this is a really long line...)